lib/adapter/playwright.js

```diff
@@ -8,6 +8,32 @@ const TestomatioClient = require('../client');
 const upload = require('../fileUploader');
 const { parseTest } = require('../util');
 
+const checkStatus = status =>
```

Comment:
what the rule causes this to move utility function on top? 
can you disable it? 
please keep these functions as `function` in the very bottom of a file

When I open file I want to see the main purpose of it, and not the functions it uses